src/build/chrome.manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.6.0",
+  "version": "1.6.2",
```

Comment:
Ahh, versioning. We're kinda using semver


Type|Desc
-- | --
MAJOR | incompatible API changes
MINOR | add functionality (backwards-compatible)
PATCH | bug fixes (backwards-compatible)

Chrome is backwards incompatible change. Should be 2.0.0
Firefox is backwards compatible, no new features. Should be 1.6.3

Let's just get these updated here, rather than a separate PR.